### PR TITLE
Make GPUImage Dependency Again

### PR DESCRIPTION
**Pull request by davidswi**
This PR adds GPUImage as a dependency to the podspec for LFLiveKit and removes the private copy of GPUImage.

### CODE REVIEW

File: LFLiveKit.podspec [hunk 1]
```diff
@@ -2,19 +2,21 @@
...
   s.source_files  = "LFLiveKit/**/*.{h,m,mm,cpp,c}"
   s.public_header_files = ['LFLiveKit/*.h', 'LFLiveKit/objects/*.h', 'LFLiveKit/configuration/*.h']
 
   s.frameworks = "VideoToolbox", "AudioToolbox","AVFoundation","Foundation","UIKit"
   s.libraries = "c++", "z"
-
+	
   s.requires_arc = true
+
+  s.dependency 'GPUImage'
```
Comment (mousetraps) [inline LFLiveKit.podspec:21]: How does CocoaPods handle versioning of dependencies?

File: LFLiveKit.podspec [hunk 2]
```diff
@@ -2,19 +2,21 @@
...
-  s.version      = "2.6+av-sync"
+  s.version      = "2.6+av-sync-gpuimage-dep"
   s.summary      = "LaiFeng ios Live. LFLiveKit."
   s.homepage     = "https://github.com/chenliming777"
   s.license      = { :type => "MIT", :file => "LICENSE" }
   s.author       = { "chenliming" => "chenliming777@qq.com" }
   s.platform     = :ios, "7.0"
   s.ios.deployment_target = "7.0"
-  s.source       = { :git => "https://github.com/mousetraps/LFLiveKit.git", :branch => "v2.6+av-sync" }
+  s.source       = { :git => "git@github.com:davidswi/LFLiveKit.git", :branch => "davids/gpuimage-as-dependency" }
```
Comment (mousetraps) [inline LFLiveKit.podspec]: This should be reverted

**Review by mousetraps — CHANGES_REQUESTED** [review]
Minor comments, and looks good after that!

File: LFLiveKit.podspec (hunk 1 above)
Comment (davidswi) [reply LFLiveKit.podspec:21]: You can add a version specifier.

Comment (davidswi) [reply LFLiveKit.podspec:21]: From Stack Overflow, a good discussion about dependencies:
https://stackoverflow.com/questions/22447062/how-do-i-create-a-cocoapods-podspec-that-has-a-dependency-that-exists-outside-of

File: LFLiveKit.podspec [hunk 3]
```diff
@@ -2,19 +2,21 @@
 Pod::Spec.new do |s|
 
   s.name         = "LFLiveKit"
-  s.version      = "2.6+av-sync"
+  s.version      = "2.6+av-sync-gpuimage-dep"
```
Comment (mousetraps) [inline LFLiveKit.podspec]: should be reverted

**Review by mousetraps — CHANGES_REQUESTED** [review]
minor comment, looks good after that!

**Pull request merged**